### ISSUE TITLE
Geo Replication on Armbian (aarch64) Bullseye fails - solved

### ISSUE THREAD

**Issue by dleonarski** [gluster/glusterdocs#716]
Hello, 
geo replication between two identical nodes:

> PRETTY_NAME="Armbian 21.08.6 Bullseye"
> NAME="Debian GNU/Linux"
> VERSION_ID="11"
> VERSION="11 (bullseye)"
> VERSION_CODENAME=bullseye
> ID=debian
> HOME_URL="https://www.debian.org/"
> SUPPORT_URL="https://www.debian.org/support"
> BUG_REPORT_URL="https://bugs.debian.org/"

> glusterfs-server/stable,now 9.2-1 arm64 [installed]

> Python 3.9.2 (default, Feb 28 2021, 17:03:44)
> [GCC 10.2.1 20210110] on linux
> 

**fails** reporting:

> Popen: command returned error [{cmd=ssh -oPasswordAuthentication=no -oStrictHostKeyChecking=no -i /var/lib/glusterd/geo-replication/secret.pem -p 22 -oControlMaster=auto -S /tmp/gsyncd-aux-ssh-xf4q5xry/c2c9e964dcee4004cd2a5b1ab33a6012.sock renegade39 /nonexistent/gsyncd slave vol renegade39::geo-vol --master-node renegade36 --master-node-id 5af54dc8-398f-481c-a120-6f2421a4abe3 --master-brick /mnt/ext/.gluster-bricks/vol --local-node renegade39 --local-node-id 86c5e793-72b0-470c-aea5-75a683fde2b1 --slave-timeout 120 --slave-log-level DEBUG --slave-gluster-log-level INFO --slave-gluster-command-dir /usr/sbin --master-dist-count 1}, {error=1}]

Manually running above ssh command causes (and discloses) Python's error (original one !). I'm not able to quote the error message now (today it works fine and yesterday's logs has gone away :-) but **it's about line #893 in /usr/lib/aarch64-linux-gnu/glusterfs/python/syncdaemon/resource .py**. Change that line as follows:

> #if not t.isAlive():
> if not t.is_alive():

In other words, "isAlive" method of the Thread class is deprecated and was replaced with "is_alive". That's it :-)

Good luck, D.

**Comment by Shwetha-Acharya** [comment]
This issue was fixed with https://github.com/gluster/glusterfs/commit/d148d1918552bfca30a003ac7cc84afde944d7b7 and is part of devel, release 10 and release 9 branches
